README.md

```diff
@@ -65,52 +65,12 @@ new KinesisWritable({
 
 `streamName` is the name of the Kinesis Stream.
 
-### Methods
-
-* `getStreamName()`: returns the stream's name.
-* `setStreamName(name)`: set the name of the stream where messages will be send.
-
 ### Events
 
-* `errorRecord`: Emitted once for each failed record at the `aws.kinesis.putRecords`'s response.
```

Comment:
Nothing in `tenant-audit` but yes in `worker-lib`.

It attempts to store it in a `fallback`. But the regular `error` listener is still compatible.